src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -32,6 +32,12 @@ public abstract partial class TestKitBase
         public object FishForMessage(Predicate<object> isMessage, TimeSpan? max = null, string hint = "")
         {
             return FishForMessage<object>(isMessage, max, hint);
+        } 
+
+        /// <inheritdoc cref="FishForMessage(Predicate{object}, TimeSpan?, string)"/>
+        public async ValueTask<object> FishForMessageAsync(Predicate<object> isMessage, TimeSpan? max = null, string hint = "")
```

Comment:
No CancellationToken support?

Reply:
It is because `ReceiveOneAsync(TimeSpan)` does not currently support that. I will need to add that in.